sys-apps/tas/tas-1.5.2.180622.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit eapi7-ver linux-info systemd
+
+CONFIG_CHECK="~IPMI_DEVICE_INTERFACE ~IPMI_HANDLER ~IPMI_SI"
```

Comment:
Please move this below standard ebuild vars.

Reply:
Done.